Tech-Journal-ConsoleApp/Tech-Journal.Test/TestingEmailEntry.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NUnit.Framework;
+using Tech_Journal_ConsoleApp;
+using System.Net.Mail;
+
+namespace Tech_Journal.Test
+{
+    [TestFixture]
+    public class TestingEmailEntry
+    {
+        [Test]
+        public void ValidateEmailInput()
```

Comment:
I would also recommend for testing of the methods to go with the naming convention of `MethodName_Should**Have This Result**On**This Condition being hit**`